observability/metric.go

```diff
@@ -0,0 +1,18 @@
+package observability
+
+import "go.uber.org/zap"
+
+var (
+	SecondsHistogramBuckets = []float64{0, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10}
+)
+
+func GetMetric[T any](metricName string, create func(metricName string) (T, error)) T {
```

Comment:
convention in Go is to avoid Get prefixes, maybe `NewMetric`? (just to avoid a non-standard prefix like Setup or Create, name doesn't have to be 100% "correct")

Reply:
good point. `New` prefix would be a better fit here since the function is responsible for instantiation of new metric object